docs/api-usage/authentication.mdx

```diff
@@ -76,25 +76,34 @@ A decoded token has the following structure:
 }
 ```
 
-You can verify tokens in your code by checking that it's signed with a trusted key. You can find the necessary JSON Web Key Set (JWKS) by visiting `https://<your-saleor-domain>/.well-known/jwks.json`.
+### Refresh token structure
 
-### Verifying the access token
+Saleor's refresh tokens are [JSON Web Tokens](https://jwt.io) signed with RS256.
 
-To verify a token, use the [`tokenVerify`](api-reference/mutations/token-verify.mdx) mutation:
+A decoded token has the following structure:
 
-```graphql {2}
-mutation {
-  tokenVerify(token: "<token>") {
-    isValid
-    errors {
-      field
-      code
-    }
-  }
+```json
+{
+  "iat": 1671039993,
+  "owner": "saleor",
+  "iss": "http://demo.saleor.io/graphql/",
+  "exp": 1673631993,
+  "token": "K3IJEFFZgshc",
+  "email": "admin@example.com",
+  "type": "refresh",
+  "user_id": "VXNlcjo1OQ==",
+  "is_staff": true,
+  "csrf_token": "cAgshsjNr9AnsMKsAkpq4Q3fb4qBBPQT5mIyDMxShQEyO1fVpwCVzVyNQAJAZfKi"
 }
 ```
 
-You can also use the [`me`](api-reference/queries/me.mdx) query to retrieve information about the currently authenticated user:
+:::info
+`csrf_token` field is deprecated and is not used in current authentication flow.
+:::
+
+### Fetching information about currently authenticated user
+
+You can use the [`me`](api-reference/queries/me.mdx) query to retrieve information about the currently authenticated user:
```

Comment:
```
You can use the [`me`](api-reference/queries/me.mdx) query to retrieve information about the currently authenticated user:
```

I know it's not the object of this PR, but could you please change the query snippet below from:
```graphql
{
  me {
    id
    email
  }
}
```

to:

```graphql
query {
  me {
    id
    email
  }
}
```

I think since we always use the `mutation` keyword in our mutation snippets, we should be consistent with queries as well. And it reads better, IMO.